scripts/run_experiment_lite.py

```diff
@@ -47,6 +47,8 @@ def run_experiment(argv):
                         help='Name of the tabular log file (in csv).')
     parser.add_argument('--text_log_file', type=str, default='debug.log',
                         help='Name of the text log file (in pure text).')
+    parser.add_argument('--tensorboard_log_dir', type=str, default='progress',
```

Comment:
perhaps the default should be or `tensorboard_summary` or `tf_summary` it is obvious what the purpose is and where it came from in the code.